README.md

```diff
@@ -41,6 +44,8 @@ Amazon Simple Notification Service (SNS) to send SMS.
 
 To get SMS notifications select newly created Notification Service in "Notif Service" drop-down on Notification admin form. OneVizion URL is automatically added to the end of the message for clarity.
 
+Please note that in order to send SMS, phone numbers must consist of a "+" character and no more than 15 digits (including the country code) or no more than 14 digits, then the country code equal to +1 will be added to the phone number. The phone number allows the use of characters "(", ")", "-".
```

Comment:
`... phone number must start with a "+" character and contain no more than 15 digits.` I do not fully understand the 2nd sentence, but probably it should say: `If the "+" character is omitted, "+1" will be added automatically`

Reply:
Fixed